<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteManage</class>
 <widget class="QMainWindow" name="RouteManage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>手机外观检测路径参数设置</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Reset">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>811</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ParamDlg">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>路径参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_VelAccDlg">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>运动参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_FrameCameraOn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>30</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DebugDlg">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>调试界面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Path_Frame_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>8.中框直边3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Path_Frame_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>10.中框直边4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Path_FrameCorner_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>5.中框R角1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Path_FrameCorner_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>7.中框R角2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Path_FrameCorner_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>9.中框R角3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Path_FrameCorner_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>11.中框R角4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CameraFocus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>13.后置摄像头</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PutPhoneOut">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>410</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>16.出口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>X轴位置：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Y轴位置：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Z轴位置：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Pos_Axis_1">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>269</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Pos_Axis_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Pos_Axis_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>中框相机：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_FocusCameraOn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>摄像头相机：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_AxisNum">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>270</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>320</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>步长：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_P2p_Length">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>310</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_P2P">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>350</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>移动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Save">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>初始化</string>
    </property>
    <widget class="QPushButton" name="pushButton_Stop">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>411</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>轨迹关键点</string>
    </property>
    <widget class="QPushButton" name="pushButton_Lisener">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>2.前置摄像头</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_PutPhoneOnAxis">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>1.预备-正面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Back">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>220</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>14.背面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Path_Frame_1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>3.中框直边1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_CylinderCamera">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>4.前听筒</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Cylinder">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>170</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>12.翻转机构</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Path_Frame_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>6.中框直边2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Flash">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>15.闪光灯</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>200</y>
      <width>381</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>运动及参数保存区</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_Vel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>41</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>速度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Acc">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>41</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>加速度：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>参数区</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>相机触发区</string>
    </property>
    <widget class="QPushButton" name="pushButton_CyliderCameraOn">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>听筒/闪光灯相机：</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox</zorder>
   <zorder>pushButton_Start</zorder>
   <zorder>pushButton_Reset</zorder>
   <zorder>plainTextEdit_Show</zorder>
   <zorder>pushButton_ParamDlg</zorder>
   <zorder>pushButton_VelAccDlg</zorder>
   <zorder>pushButton_FrameCameraOn</zorder>
   <zorder>pushButton_DebugDlg</zorder>
   <zorder>pushButton_Path_Frame_3</zorder>
   <zorder>pushButton_Path_Frame_4</zorder>
   <zorder>pushButton_Path_FrameCorner_1</zorder>
   <zorder>pushButton_Path_FrameCorner_2</zorder>
   <zorder>pushButton_Path_FrameCorner_3</zorder>
   <zorder>pushButton_Path_FrameCorner_4</zorder>
   <zorder>pushButton_CameraFocus</zorder>
   <zorder>pushButton_PutPhoneOut</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_Pos_Axis_1</zorder>
   <zorder>lineEdit_Pos_Axis_2</zorder>
   <zorder>lineEdit_Pos_Axis_3</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_FocusCameraOn</zorder>
   <zorder>label_5</zorder>
   <zorder>comboBox_AxisNum</zorder>
   <zorder>label_6</zorder>
   <zorder>lineEdit_P2p_Length</zorder>
   <zorder>pushButton_P2P</zorder>
   <zorder>pushButton_Save</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
